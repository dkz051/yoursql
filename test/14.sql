CREATE DATABASE qAz;
CREATE DATABASE eDc;
CREATE DATABASE iJn;
CREATE DATABASE aCs;
SHOW DATABASES;
USE qAz;
CREATE TABLE qwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
CREATE TABLE sqwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (-1109, 1583, -334, 6313);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (-1578, 154, -1534, 3512);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (-2183, -2013, -2139, 1599);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1484, -1421, -279, 2689);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (-2182, -1592, 403, 1809);
UPDATE qwe SET rty=-1578 WHERE -2010>ert AND 1873<wer AND wer=rty OR wer<rty AND tyu<1302 AND wer<rty;
SELECT wer FROM qwe WHERE ert>tyu AND ert>tyu AND rty<rty OR rty<wer AND -1562<rty AND rty<-2093;
UPDATE qwe SET wer=-887 WHERE tyu<tyu AND tyu>ert AND ert<wer OR rty>tyu AND rty<1172 AND tyu>-106;
SELECT wer FROM qwe WHERE ert<ert AND rty>rty AND wer>wer OR -1156>tyu AND ert<ert AND wer<wer;
UPDATE qwe SET wer=-1524 WHERE wer<ert AND -418<ert AND 673<wer OR ert<wer AND ert>tyu AND wer=wer;
SELECT ert FROM qwe WHERE 836>-1706 AND ert>-274 AND tyu>tyu OR ert<rty AND rty>ert AND wer>wer;
UPDATE qwe SET ert=1404 WHERE wer>wer AND rty<rty AND 710>ert OR rty<wer AND rty<wer AND rty>rty;
SELECT ert FROM qwe WHERE rty=tyu AND wer<rty AND tyu<tyu OR rty<wer AND tyu>ert AND rty>348;
UPDATE qwe SET rty=-128 WHERE ert>448 AND wer>ert AND rty>tyu OR ert<wer AND tyu>1472 AND tyu<tyu;
SELECT ert FROM qwe WHERE tyu>-1857 AND rty>wer AND rty<tyu OR wer>ert AND rty<tyu AND tyu>1234;
SHOW columns from sqwe;
DROP DATABASE aCs;
DROP DATABASE qAz;
DROP DATABASE eDc;
DROP DATABASE iJn;
