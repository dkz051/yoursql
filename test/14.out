Database
aCs
eDc
iJn
qAz
ert
-2013
-1592
-1421
154
1583
Field	Type	Null	Key	Default	Extra
wer	int(11)	YES		NULL	
ert	int(11)	YES		NULL	
rty	int(11)	YES		NULL	
tyu	int(11)	NO	PRI	NULL	
